Testscripts/Linux/utils.sh

```diff
@@ -3489,4 +3489,59 @@ function CreateFile()
 	else
 		LogMsg "Error: $file_path failed to create with size $size"
 	fi
+}
+
+# Check available packages
+function check_package ()
+{
+	local package_list=("$@")
+	for package_name in "${package_list[@]}"; do
+		case "$DISTRO_NAME" in
+			oracle|rhel|centos)
+				yum provides "$package_name" | grep -i Matched
+				return $?
+				;;
+
+			ubuntu|debian)
+				apt-cache policy "$package_name" | grep Candidate
+				return $?
+				;;
+
+			suse|opensuse|sles)
+				zypper search "$package_name"
+				return $?
+				;;
+
+			clear-linux-os)
+				swupd search "$package_name" | grep -v "failed"
+				return $?
+				;;
+			*)
+				echo "Unknown distribution"
+				return 1
+		esac
+	done
+}
+
+
+# Install nvme
+function install_nvme()
```

Comment:
should be install_nvme_cli?

Reply:
Thats Correct .. Changed function name to install_nvme_cli